### PR TITLE
DP-1334 Improve documentation

### PR DESCRIPTION
**Pull request by lily-es**
This PR improves the documentation, the makefile, removes unnecessary files, and adds a new counter for the count of errors when posting events.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # dataplatform-probe
-Monitoring service for dataplatform services and events
+Monitoring service for dataplatform services and events
```
Comment (simenheg) [inline README.md]: Some trailing whitespace snuck in here.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # dataplatform-probe
-Monitoring service for dataplatform services and events
+Monitoring service for dataplatform services and events  
 
-## Running
+This application continuously sends events to the dataplatform pipeline to test the latency of the pipeline.
```
Comment (simenheg) [inline README.md:4]: And here.

File: README.md [hunk 3]
```diff
@@ -18,16 +20,43 @@ Running this app requires these environment variables to be set:
 | EVENT_INTERVAL_SECONDS | Number of seconds to wait before sending an event                   |
 | MAX_CONSECUTIVE_ERRORS | Number of consecutive errors allowed when attempting to post events |
 
-Once these environment variables are set you can
-run the app with [Pipenv](https://github.com/pypa/pipenv):
-1. `pipenv install`
-2. `pipenv run app`
+The [run.sh](run.sh) script provided sets these environment variables and starts the app. If you wish to use it,
+make sure to edit it to include the missing environment variables(secrets and such)
```
Comment (simenheg) [inline README.md]: Missing whitespace before the opening parenthesis (and punctuation at the end).

File: README.md [hunk 4]
```diff
@@ -18,16 +20,43 @@ Running this app requires these environment variables to be set:
...
+`make lint`
+#### Run the app
+`make run-app`
+#### Execute the [run.sh](run.sh) script
+`make run-script`
 
 ## Metrics
 This app uses the [prometheus_client](https://github.com/prometheus/client_python) library to expose
-metrics regarding pipeline latency through a http server on port 8000
+metrics to prometheus regarding pipeline latency through a http server on port 8000.
```
Comment (simenheg) [inline README.md]: Trailing spaces.

Comment (simenheg) [inline README.md]: prometheus => Prometheus

File: README.md [hunk 5]
```diff
@@ -18,16 +20,43 @@ Running this app requires these environment variables to be set:
...
 ## Metrics
 This app uses the [prometheus_client](https://github.com/prometheus/client_python) library to expose
-metrics regarding pipeline latency through a http server on port 8000
+metrics to prometheus regarding pipeline latency through a http server on port 8000.  
+
+These metrics are:
+- **events_posted**(Counter): The count of events posted to the pipeline
+- **event_post_errors**(Counter): The count of errors experienced when posting events
+- **events_received**(Counter): The count of events received from the pipeline
+- **wrong_appid**(Counter): The count of events received with the wrong App ID(posted by another instance of probe)
```
Comment (simenheg) [inline README.md]: Missing whitespace before opening parenthesis.

**Review by simenheg — APPROVED** [review]
Very nice!

Just a few formatting nitpicks.

File: README.md (hunk 2 above)
Comment (lily-es) [reply README.md:4]: Yeah here its on purpose to cause the new sentence to show below

**Pull request merged**